br/pkg/lightning/backend/local/local.go

```diff
@@ -1400,6 +1420,27 @@ func (*Backend) isRetryableImportTiKVError(err error) bool {
 	return common.IsRetryableError(err)
 }
 
+func checkDiskAvail(ctx context.Context, store *pdhttp.StoreInfo) error {
```

Comment:
Why here we need checkDiskAvail？

Reply:
these logic is moved from old file line 1422~1430, without the retry logic, because retry is handled inside PD HTTP client